lib/sycamore/sycamore/connectors/elasticsearch/elasticsearch_writer.py

```diff
@@ -32,16 +32,18 @@ class ElasticTargetParams(BaseDBWriter.TargetParams):
             }
         }
     )
-    flatten_properties: bool = False
+    wait_for_completion: str = "false"
```

Comment:
this one prob doesn't need to be a target param. I'm pretty sure the "wait for all actions before refreshing" option is always correct

Reply:
Oh I see you use it in places in the test. ok, nvm